src/vstest.console/CommandLine/AssemblyMetadataProvider.cs

```diff
@@ -32,7 +32,7 @@ internal AssemblyMetadataProvider(IFileHelper fileHelper)
     }
 
     /// <inheritdoc />
-    public FrameworkName GetFrameWork(string filePath)
+    public FrameworkName GetFrameworkName(string filePath)
```

Comment:
Somehow unrelated but I got confused by the API name compared to its return type when fixing the auto-detection algorithm so I also fixed the name.